lib/pacemaker/pcmk_sched_primitive.c

```diff
@@ -1014,9 +1014,19 @@ pcmk__primitive_apply_coloc_score(pe_resource_t *dependent,
     }
 }
 
+/*!
+ * \internal
+ * \brief Return action flags for a given primitive resource action
+ *
+ * \param[in,out] action  Action to get flags for
+ * \param[in]     node    If not NULL, limit effects to this node
```

Comment:
I've gone back and forth between documenting each implementation fully vs just adding a comment like 

    // Primitive implementation of resource_alloc_functions_t:action_flags()

If we are documenting them individually, then it does make sense to put "ignored" here. I think the return documentation is still accurate even if it's just action->flags for primitives since that's what's appropriate in that case.